lang/en/admin.php

```diff
@@ -793,8 +793,10 @@
 $string['manageformats'] = 'Manage course formats';
 $string['manageformatsgotosettings'] = 'Default format can be changed in {$a}';
 $string['managelang'] = 'Manage';
-$string['questionbanks'] = 'Question bank plugins';
-$string['manageqbanks'] = 'Manage question bank plugins';
+$string['questionbanks'] = 'Question banks';
+$string['columnsortorder'] = 'Column sort order';
+$string['qbanksortdisclaimer'] = 'Column order modification in this page will reorder column display in question bank view';
```

Comment:
'columnsortorder' and 'qbanksortdisclaimer' should be move to lang/en/question as is an external page the is in core_question